programs/mango-v4/src/instructions/serum3_place_order.rs

```diff
@@ -204,7 +204,7 @@ pub fn serum3_place_order(
     ctx: Context<Serum3PlaceOrder>,
     side: Serum3Side,
     limit_price: u64,
-    max_base_qty: u64,
+    mut max_base_qty: u64,
```

Comment:
I feel weird about mut'ing this here, wdyt?

Reply:
ok with it